### PR TITLE
Adding List Teams API documentation

### PR DESCRIPTION
**Pull request by akhilkohlimicrosoft**

### CODE REVIEW

File: api-reference/beta/api/teams-list.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+doc_type: apiPageType
+---
+
+# List Teams
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Get list of all [teams](../resources/team.md) in an organization.
```
Comment (isvargasmsft) [inline api-reference/beta/api/teams-list.md:16]: Description in lines 3 and 16: Please rephrase to:
List all [teams] (link here) in an organization.

File: api-reference/beta/api/teams-list.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
+---
+title: "List Teams in an organization"
+description: "List all the teams available in an organization"
+author: "akhilkohlimicrosoft"
+ms.localizationpriority: high
+ms.prod: "microsoft-teams"
+doc_type: apiPageType
+---
+
+# List Teams
```
Comment (isvargasmsft) [inline api-reference/beta/api/teams-list.md:10]: Title in lines 2 and 10: please use small caps for teams:
**List teams**

File: api-reference/beta/api/teams-list.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+
+## Examples
+
+### Example 1: Get a list of teams
+
+#### Request
+
+The following is an example of the request.
+
+# [HTTP](#tab/http)
```
Comment (isvargasmsft) [inline api-reference/beta/api/teams-list.md:61]: Line 61: Please delete this line. Snippets are automatically generated. Details here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103162/Generate-code-snippets

File: api-reference/beta/api/teams-list.md [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+<!-- {
+  "blockType": "request",
+  "name": "get_teams"
+}-->
+```msgraph-interactive
```
Comment (isvargasmsft) [inline api-reference/beta/api/teams-list.md:66]: Line 66: please replace **msgraph-interactive** for **http**.

File: api-reference/beta/api/teams-list.md [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+GET https://graph.microsoft.com/beta/teams
+```
+---
+
+### Response
```
Comment (isvargasmsft) [inline api-reference/beta/api/teams-list.md:71]: Line 71: Please use H4 Response:

#### Response

File: api-reference/beta/api/teams-list.md [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+The following example shows the response.
```
Comment (isvargasmsft) [inline api-reference/beta/api/teams-list.md:72]: Line 74: Please rephrase to:
The following is an example of a response.

File: api-reference/beta/api/teams-list.md [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+
+>**Note:** The response object shown here might be shortened for readability.
+<!-- {
+  "blockType": "response",
+  "truncated": true,
+  "@odata.type": "microsoft.graph.group",
```
Comment (isvargasmsft) [inline api-reference/beta/api/teams-list.md:78]: Line 78: In line 48 we state that this method return a collection of team objects. However, we are telling API Doctor that this return a collection of groups. 

Can you please clarify?

File: api-reference/beta/api/teams-list.md (hunk 3 above)
Comment (isvargasmsft) [reply api-reference/beta/api/teams-list.md:61]: Please apply to other instances in this PR.

File: api-reference/beta/api/teams-list.md (hunk 4 above)
Comment (isvargasmsft) [reply api-reference/beta/api/teams-list.md:66]: Please apply to other files and instances in this PR

File: api-reference/beta/api/teams-list.md [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+<!-- {
+  "blockType": "request",
+  "name": "get_teams"
+}-->
+```msgraph-interactive
+GET https://graph.microsoft.com/beta/teams?$filter=startswith(displayName, 'A')&$top=2
+```
+---
+
+### Response
```
Comment (isvargasmsft) [inline api-reference/beta/api/teams-list.md:123]: H4 please

File: api-reference/beta/api/teams-list.md [hunk 9]
```diff
@@ -0,0 +1,195 @@
...
+The following example shows the response.
```
Comment (isvargasmsft) [inline api-reference/beta/api/teams-list.md:124]: Please rephrase to: The following is an example of a response.

File: api-reference/beta/toc.yml [hunk 10]
```diff
@@ -551,6 +551,8 @@ items:
           href: api/userteamwork-teamsappinstallation-upgrade.md
         - name: Remove app for user
           href: api/userteamwork-delete-installedapps.md
+        - name: Get list of teams in an organization
```
Comment (isvargasmsft) [inline api-reference/beta/toc.yml:554]: Please rephrase to: List all teams in an organization

Comment (isvargasmsft) [inline api-reference/beta/toc.yml:554]: Missing in this PR:
Changelog entry for the new operation (list teams)

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello Akhil, 

Thank you very much for your work in this PR. 

Please add the proper changelog entry for the new operation, and please review a few comments on structure and phrasing. 

Feel free to ping me on the side if you have any questions. 

Kind regards, 

Isaac Vargas

File: api-reference/beta/api/teams-list.md [hunk 11]
```diff
@@ -0,0 +1,195 @@
...
+
+## HTTP request
+<!-- { "blockType": "ignored" } -->
+```http
+GET /teams
+```
+
+## Optional query parameters
+
+This method supports the  `$filter`, `$select`, `$top`, `$skiptoken`, `$count` [OData query parameters](/graph/query-parameters) to help customize the response.
```
[the remark this answers is not in the document]
Comment (akhilkohlimicrosoft) [reply api-reference/beta/api/teams-list.md:35]: Added property / value. Please check if this is fine. Thanks

[the remark this answers is not in the document]
Comment (akhilkohlimicrosoft) [reply api-reference/beta/api/teams-list.md:35]: I have provided a link to OData Query parameters which explains the property supported for each query parameter. Please let me know if this works. Thanks

**Review by isvargasmsft — APPROVED** [review]

File: api-reference/beta/toc.yml [hunk 12]
```diff
@@ -551,6 +551,8 @@ items:
           href: api/userteamwork-teamsappinstallation-upgrade.md
         - name: Remove app for user
           href: api/userteamwork-delete-installedapps.md
+        - name: List all teams in an organization
```
Comment (Lauragra) [inline api-reference/beta/toc.yml:554]: This is the wrong location in the TOC to surface this API. This is the Teamwork node under Users; we should only surface user-related APIs here. Please move this TOC entry to the Teamwork node, under Team.

**Review by isvargasmsft — APPROVED** [review]

**Pull request merged**